admin_manual/configuration_database/mysql_4byte_support.rst

```diff
@@ -31,15 +37,33 @@ Now you should be able to use Emojis in your file names, calendar events, commen
 MariaDB support
 ---------------
 
-.. note:: This is even more **experimental**.
+MariaDB 10.3 or later
+=====================
+1. Make sure the following InnoDB settings are set on your MariaDB server::
+
+    [mysqld]
+    innodb_file_per_table=1
+
+2. Continue at step 2 of the MySQL instructions.
+
+
+MariaDB 10.2 or earlier
+=======================
+
+1. Make sure the following InnoDB settings are set on your MySQL server::
+
+    [mysqld]
+    innodb_large_prefix=true
+    innodb_file_format=barracuda
+    innodb_file_per_table=1
 
-1. Follow MySQL steps 1, 2 and 3
+2. Restart the MariaDB server in case you changed the configuration in step 1.
 
-2. Figure out whether the file format was changed to Barracuda::
+3. Figure out whether the file format was changed to Barracuda::
 
     MariaDB> SELECT NAME, SPACE, FILE_FORMAT FROM INFORMATION_SCHEMA.INNODB_SYS_TABLES WHERE NAME like "nextcloud%";
 
-If the file format is "Barracuda" for every single table, nothing is left to do. Continue with the MySQL instructions. While testing, all tables' file format was "Antelope".
+If the file format is "Barracuda" for every single table, nothing special is left to do. Continue with the MySQL instructions at step 3. While testing, all tables' file format was "Antelope".
 
 3. The tables needs to be migrated to "Barracuda" manually, one by one. SQL commands can be created easily, however::
```

Comment:
damn, twice 3.

Reply:
sorry, this slipped through. Wel, in md it wouldn't matter^^